src/models/payment.js

```diff
@@ -24,6 +24,23 @@ module.exports = (sequelize, DataTypes) => {
         type: DataTypes.STRING,
         allowNull: true,
       },
+      statusPayment: {
```

Comment:
Название модели не нужно в поле добавлять,  при использовании Paymet.status лучше читаться  будет чем 
Paymet. statusPayment
```suggestion
      status: {
```

Reply:
исправлено